crates/protect-ffi/src/encrypt_config.rs

```diff
@@ -158,7 +158,7 @@ impl FromStr for EncryptConfig {
 }
 
 impl EncryptConfig {
-    pub fn to_config_map(self) -> HashMap<Identifier, ColumnConfig> {
+    pub fn into_config_map(self) -> HashMap<Identifier, ColumnConfig> {
```

Comment:
Fixing up a Clippy warning here. Clippy expects the method to be prefixed with `into` since it consumes self. See: https://rust-lang.github.io/rust-clippy/master/index.html#wrong_self_convention.